Makefile

```diff
@@ -1,82 +1,32 @@
 #
-# Makefile for phoenix-rtos-usb
+# Makefile for phoenix-rtos-posixsrv
 #
-# Copyright 2019 Phoenix Systems
+# Copyright 2019, 2021 Phoenix Systems
 #
 # %LICENSE%
 #
 
 SIL ?= @
 MAKEFLAGS += --no-print-directory
 
-#TARGET ?= ia32-qemu
-#TARGET ?= armv7-stm32
-TARGET ?= arm-imx6ull
 
-TOPDIR := $(CURDIR)
-PREFIX_BUILD ?= ../_build/$(TARGET)
-PREFIX_BUILD := $(abspath $(PREFIX_BUILD))
-BUILD_DIR ?= $(PREFIX_BUILD)/$(notdir $(TOPDIR))
-BUILD_DIR := $(abspath $(BUILD_DIR))
+include ../phoenix-rtos-build/Makefile.common
+include ../phoenix-rtos-build/Makefile.$(TARGET_SUFF)
 
-# Compliation options for various architectures
-TARGET_FAMILY = $(firstword $(subst -, ,$(TARGET)-))
-include Makefile.$(TARGET_FAMILY)
+CFLAGS += $(BOARD_CONFIG)
 
-# build artifacts dir
-CURR_SUFFIX := $(patsubst $(TOPDIR)/%,%,$(abspath $(CURDIR))/)
-PREFIX_O := $(BUILD_DIR)/$(CURR_SUFFIX)
 
-# target install paths, can be provided exterally
-PREFIX_A ?= $(PREFIX_BUILD)/lib/
-PREFIX_H ?= $(PREFIX_BUILD)/include/
-PREFIX_PROG ?= $(PREFIX_BUILD)/prog/
-PREFIX_PROG_STRIPPED ?= $(PREFIX_BUILD)/prog.stripped/
-
-CFLAGS += -I"$(PREFIX_H)"
-LDFLAGS += -L"$(PREFIX_A)"
-
-# add include path for auto-generated files
-CFLAGS += -I"$(BUILD_DIR)/$(CURR_SUFFIX)"
-
-ARCH =  $(SIL)@mkdir -p $(@D); \
-	(printf "AR  %-24s\n" "$(@F)"); \
-	$(AR) $(ARFLAGS) $@ $^ 2>/dev/null
-
-LINK = $(SIL)mkdir -p $(@D); \
-	(printf "LD  %-24s\n" "$(@F)"); \
-	$(LD) $(LDFLAGS) -o "$@"  $^ $(PREFIX_A)libtty.a $(LDLIBS)
-	
-HEADER = $(SIL)mkdir -p $(@D); \
-	(printf "HEADER %-24s\n" "$<"); \
-	cp -pR "$<" "$@"
-
-$(PREFIX_O)%.o: %.c
-	@mkdir -p $(@D)
-	$(SIL)(printf "CC  %-24s\n" "$<")
-	$(SIL)$(CC) -c $(CFLAGS) "$<" -o "$@"
-	$(SIL)$(CC) -M  -MD -MP -MF $(PREFIX_O)$*.c.d -MT "$@" $(CFLAGS) $<
-
-$(PREFIX_O)%.o: %.S
-	@mkdir -p $(@D)
-	$(SIL)(printf "ASM %s/%-24s\n" "$(notdir $(@D))" "$<")
-	$(SIL)$(CC) -c $(CFLAGS) "$<" -o "$@"
-	$(SIL)$(CC) -M  -MD -MP -MF $(PREFIX_O)$*.S.d -MT "$@" $(CFLAGS) $<
-	
-$(PREFIX_PROG_STRIPPED)%: $(PREFIX_PROG)%
-	@mkdir -p $(@D)
-	@(printf "STR %-24s\n" "$(@F)")
-	$(SIL)$(STRIP) -o $@ $<
-	
-all: $(PREFIX_PROG_STRIPPED)posixsrv
-
-$(PREFIX_PROG)posixsrv: $(addprefix $(PREFIX_O), event.o pipe.o pty.o special.o tmpfile.o posixsrv.o)
-	$(LINK)
-	
 .PHONY: clean
```

Comment:
I think we can delete `clean` due to https://github.com/phoenix-rtos/phoenix-rtos-build/blob/4c9653393f8d2974e2c084275dabd175243cfc94/build.sh#L91

Reply:
Done.